Nos/Views/Discover/FeaturedAuthor.swift

```diff
@@ -12,6 +12,21 @@ struct FeaturedAuthor {
     /// The categories in which to show the author on the Discover tab.
     /// No need to include the `.all` category; that's done automatically.
     let categories: [FeaturedAuthorCategory]
+    
+    let rawID: RawAuthorID
+    
+    internal init(name: String, npub: String, categories: [FeaturedAuthorCategory]) {
```

Comment:
```suggestion
    init(name: String, npub: String, categories: [FeaturedAuthorCategory]) {
```

`internal` isn't needed, right?

Reply:
yeah, it was auto-generated.